WobblyStealthGame/Assets/Scripts/Audio/VolumeController.cs

```diff
@@ -67,10 +67,22 @@ public void UpdateVolumeAmbient()
 
     void Load()
     {
-        masterSlider.value = PlayerPrefs.GetFloat("VolumeMaster");
-        effectsSlider.value = PlayerPrefs.GetFloat("VolumeEffects");
-        musicSlider.value = PlayerPrefs.GetFloat("VolumeMusic");
-        ambientSlider.value = PlayerPrefs.GetFloat("VolumeAmbient");
+        if (PlayerPrefs.HasKey("VolumeMaster"))
+        {
+            masterSlider.value = PlayerPrefs.GetFloat("VolumeMaster");
+        }
+        if (PlayerPrefs.HasKey("VolumeEffects"))
+        {
+            effectsSlider.value = PlayerPrefs.GetFloat("VolumeEffects");
+        }
+        if (PlayerPrefs.HasKey("VolumeMusic"))
+        {
+            musicSlider.value = PlayerPrefs.GetFloat("VolumeMusic");
+        }
+        if (PlayerPrefs.HasKey("VolumeAmbient"))
+        {
+            ambientSlider.value = PlayerPrefs.GetFloat("VolumeAmbient");
+        }
```

Comment:
Je kan gewoon een standaard waarde mee geven aan de player prefs, deze wordt gebruikt als er geen waarde bestaat. [Meer informatie over de standaard waarde(unity docs) ](https://docs.unity3d.com/ScriptReference/PlayerPrefs.GetFloat.html)

```suggestion
        masterSlider.value = PlayerPrefs.GetFloat("VolumeMaster", 1.0f);
        effectsSlider.value = PlayerPrefs.GetFloat("VolumeEffects", 1.0f);
        musicSlider.value = PlayerPrefs.GetFloat("VolumeMusic", 1.0f);
        ambientSlider.value = PlayerPrefs.GetFloat("VolumeAmbient", 1.0f);
```